src/libs/OptionsListUtils.js

```diff
@@ -140,14 +161,16 @@ Onyx.connect({
  *
  * @param {String} searchValue
  * @param {String} searchText
+ * @param {Object} participantNames
  * @returns {Boolean}
  */
-function isSearchStringMatch(searchValue, searchText) {
+function isSearchStringMatch(searchValue, searchText, participantNames) {
     const searchWords = searchValue.split(' ');
     return _.every(searchWords, (word) => {
         const matchRegex = new RegExp(Str.escapeForRegExp(word), 'i');
         const valueToSearch = searchText && searchText.replace(new RegExp(/&nbsp;/g), '');
-        return matchRegex.test(valueToSearch);
+        return matchRegex.test(valueToSearch)
+            || participantNames[word.trim().replace(new RegExp(/,/g), '')];
```

Comment:
Maybe it would be better to replace all `','` with `' '` in `searchValue` before we split it into words so e.g.
`marc,jasper,scott` would become `['marc', 'jasper', 'scott']` instead of `['marcjasperscott']`?